R/helpers.R

```diff
@@ -1758,3 +1785,41 @@ gt_latex_dependencies <- function() {
          call. = FALSE)
   }
 }
+
+#' Helper function for specifying a font from the Google Fonts service
+#'
+#' The `google_fonts()` helper function can be used wherever a font name should
+#' be specified. There are two instances where this helper can be used: the
+#' `name` argument in [opt_table_font()] (for setting a table font) and in that
+#' of [cell_text()] (used with [tab_style()]).
+#'
+#' @param name The complete name of a font available in Google Fonts.
+#'
+#' @return An object of class `google_fonts`.
+#'
+#' @family Helper Functions
+#' @section Function ID:
+#' 7-22
+#'
+#' @export
+google_fonts <- function(name) {
+
+  import_stmt <-
+    name %>% tidy_gsub(" ", "+") %>%
+    paste_between(
+      c(
+        "@import url('https://fonts.googleapis.com/css2?family=",
+        ":ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');"
+      )
+    )
+
+  google_fonts <-
+    list(
+      name = name,
+      import_stmt = import_stmt
+    )
+
+  class(google_fonts) <- "google_fonts"
```

Comment:
Rather than `google_fonts` it'd be nice if this was something more generic, since it could be used for any combination of font name + CSS.

Reply:
Class renamed to `"font_css"`.